### DISCUSSION TITLE
Extension Launcher but without bookmarklets?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1061] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there a way to download stuff like the recovery utility for flashing for downgrading while it's blocked by admin?

**Comment by Blobby-Boi** [comment, marked as answer]
If you are in 115-120 you can do skiovox

**Reply by FunnyCorgi** [reply]
but then that defeats the whole point of skiovox. I am asking about it for my main account(if its even possible).
